applicationconfig/client/pom.xml

```diff
@@ -0,0 +1,67 @@
+<!--
+ Copyright (c) Microsoft Corporation. All rights reserved.
+ Licensed under the MIT License. See License.txt in the project root for
+ license information.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <relativePath>../../pom.client.xml</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-app-configuration</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+
+  <name>Microsoft Azure SDK component for Application Configuration</name>
+  <description>This package contains the Microsoft Azure Application Configuration SDK component.</description>
```

Comment:
Didn't these get updated to 'client library' at some point?

Reply:
It's in the samples PR. Once I merge it, it'll be updated.